R/estimate.R

```diff
@@ -0,0 +1,128 @@
+#'  Source code for the ESTIMATE algorithm:
+#'  Estimate of Stromal and Immune Cells in Malignant Tumor Tissues from Expression Data
+#'  (https://doi.org/10.1038/ncomms3612)
+#'  Source: http://r-forge.r-project.org/projects/estimate/
+#'  Copyright: 2013-2022, MD Anderson Cancer Center (MDACC)
+#'  License: GPL-2
+#'
+#'  Source code adapted from: http://download.r-forge.r-project.org/bin/windows/contrib/3.3/estimate_1.0.13.zip
+#'
+
+
+
+#'  ESTIMATE algortihm: This function computes stromal, immune, and ESTIMATE scores
+#'  per sample using gene expression data, through GSEA. The ESTIMATE score is used
+#'  to compute an estimate of the tumor purity
+#'  *Warning*: The tumor purity estimation was originally intended only for microarray
+#'  affymetrix data
```

Comment:
does this mean we should do any transformation to the data by default? 

immundeconv expects untransformed TPM as default input.